### PR TITLE
Resolves issue #4

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- Resolves issue #4.
- Fix coding style via `PHP_CodeSniffer` tool.
- Fix some code snippets via `PHPStan` tool.

### CODE REVIEW

File: src/Processor/Symfony.php [hunk 1]
```diff
@@ -31,11 +33,11 @@ class Symfony implements Processor
      * @param  int[]  $acceptableExitCodes
      * @return \SebastianFeldmann\Cli\Command\Result
      */
-    public function run(string $cmd, array $acceptableExitCodes = [0]) : Result
+    public function run(string $cmd, array $acceptableExitCodes = [0]): Result
     {
         $process = method_exists('Symfony\Component\Process\Process', 'fromShellCommandline')
                  ? Process::fromShellCommandline($cmd)
-                 : new Process($cmd);
+                 : new Process([$cmd]);
```
[the remark this answers is not in the document]
Comment (peter279k) [reply src/Processor/Symfony.php:40]: @sebastianfeldmann, this change is effected by the PHPStan tool.

To revert this issue, maybe we can tell PHPStan tool to ignore this parameter type check?

[the remark this answers is not in the document]
Comment (peter279k) [reply src/Processor/Symfony.php:40]: @markvds, thanks for your reply and report, this change is effected by `PHPstan` tool, you should track this comment :).

[the remark this answers is not in the document]
Comment (sebastianfeldmann) [reply src/Processor/Symfony.php:40]: I put the error on the PHPStan ignore list for now. Maybe we can come up with something more elegant.
PHPStan supports an annotation @ignoreNextLine maybe we can use that one instead. I will investigate and see what I can come up with ;)

[the remark this answers is not in the document]
Comment (markvds) [reply src/Processor/Symfony.php:40]: Personally, I would annotate the code and put a comment to explain why it's okay to call the constructor with a string as first argument. It prevents the same mistake in the future; IDEs can also complain about the type mismatch and someone might someday think it's a good idea to correct the type mismatch :)

**Pull request merged**